solidity/contracts/bridge/Deposit.sol

```diff
@@ -85,6 +85,7 @@ library Deposit {
         // Deposit amount in satoshi.
         uint64 amount;
         // UNIX timestamp the deposit was revealed at.
+        // XXX: unsigned 32-bit integer unix seconds, will break around 2106
```

Comment:
Let's be specific that it's about `the year 2016`, as it's unclear what the number `2016` means. Also, it will happen not `around 2016` but `in 2016` (on Sunday, 7 February 2106).

```suggestion
        // XXX: unsigned 32-bit integer unix seconds, will break in the year 2106
```

Reply:
48b54c8bebdaca20defc8d6c986c8102ead443a6